File: app/src/main/java/kittentrate/score/PlayerScore.java
package kittentrate.score;

/**
 * Created by Manuel Lorenzo on 23/03/2017.
 */

public class PlayerScore {
    private String playerName;
    private int playerScore;

    public PlayerScore(String playerName, int score) {
        this.playerName = playerName;
        this.playerScore = score;
    }

    public String getPlayerName() {
        return playerName;
    }

    public void setPlayerName(String playerName) {
        this.playerName = playerName;
    }

    public int getPlayerScore() {
        return playerScore;
    }

    public void setPlayerScore(int playerScore) {
        this.playerScore = playerScore;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;

        PlayerScore that = (PlayerScore) o;

        return playerScore == that.playerScore && playerName.equals(that.playerName);
    }

    @Override
    public int hashCode() {
        int result = playerName.hashCode();
        result = 31 * result + playerScore;
        return result;
    }
}

File: app/src/main/java/kittentrate/ui/score/PlayerScore.kt
package kittentrate.ui.score

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

/**
 * Created by Manuel Lorenzo on 23/03/2017.
 */
@Entity(tableName = "scores")
data class PlayerScore(@field:ColumnInfo(name = "name") var playerName: String,
                       @field:ColumnInfo(name = "score") var playerScore: Long) {
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo
    lateinit var id: Integer
}
